src/Pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {Meta, Story} from '@storybook/react'
+import {ComponentProps} from '../utils/types'
+import Pagination from './Pagination'
+
+export default {
+  title: 'Components/Pagination',
```

Comment:
Could you add "Pagination" to the component list in storybook.test.ts? https://github.com/primer/react/blob/main/src/__tests__/storybook.test.tsx